content/news/2023-01-20-galaxy-sc/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: A new tool for multiplexed single-cell RNA-seq analysis in Galaxy
+date: '2023-01-20'
+tags: [tools]
+subsites: [global]
+---
+
+More and more labs are generating single-cell-RNA-seq data and would like to analyze them in Galaxy.
+Recently, [10x Genomics](https://www.10xgenomics.com/) proposed a kit (CellPlex) to allow multiplexing up to 12 samples in the same Chromium run,
+reducing drastically the cost of a single experiment. With this new technique, the user sequences two libraries: a library containing the
+mRNA and a library containing the Cell Multiplexing Oligos (CMOs) asigning each cell barcode to a sample.
+The analysis of these datasets can be done with CellRanger, the software developed by 10x Genomics which is not open-source. An open-source alternative
+is [STARsolo](https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md), this tool is available in Galaxy since 2019 and regularly updated.
+A [tutorial](https://training.galaxyproject.org/training-material/topics/single-cell/tutorials/scrna-preprocessing-tenx/tutorial.html) in the
+GTN demonstrates how to analyze the FASTQ files containing the mRNA. However, this tool does not handle the second library generated from CellPle
+datasets for which another analysis need to be done. An open-source software [CITE-seq-Count](https://hoohm.github.io/CITE-seq-Count/) can fill the gap - and is now also avillbe in Galaxy.
+
+CITE-seq-Count allows to analyze the second set of FASTQ files containing the CMOs. Providing the FASTQs, CITE-seq-Count will output the number of
+CMO for each cell barcode, allowing to filter empty cells, doblets and asign each cell barcode to the corresponding sample.
```

Comment:
For post-processing, you can then use the normal candidates like scanpy and co? Maybe we can add this?

Reply:
Yes but outside of Galaxy... Should I mention them?

Reply:
Oh I thought we could use the tools inside Galaxy. What is missing in Galaxy?

Reply:
For an analysis with scanpy, you need hashsolo: https://scanpy.readthedocs.io/en/latest/generated/scanpy.external.pp.hashsolo.html
For an analysis with Seurat, you need HTODemux: https://satijalab.org/seurat/reference/htodemux
I am not familiar with analysis with scanpy which seems to be the standard in galaxy. Maybe @nomadscientist could help.

Reply:
Ok, then we leave it like that and see if we can add one or both tools.

Let me know when you think this PR is out of WIP.

Reply:
Or we wait for the tools to be implemented...

Reply:
Up to you...

Reply:
> For an analysis with scanpy, you need hashsolo: https://scanpy.readthedocs.io/en/latest/generated/scanpy.external.pp.hashsolo.html For an analysis with Seurat, you need HTODemux: https://satijalab.org/seurat/reference/htodemux I am not familiar with analysis with scanpy which seems to be the standard in galaxy. Maybe @nomadscientist could help.

Thanks for working on CITE-Seq-count! Yes, hashsolo and/or htodemux are to be implemented. It should be easy to filter an anndata object after running hashsolo. I'm not sure how easy it is to filter or subset a seurat object after running HTODemux. This is something to be considered while implementing the HTODemux tool. Otherwise, both of them seem to be simple functions. What we need is their corresponding test data. The test data provided in their git repos are huge. If you know/have some tiny test data sets please point me to them. I will integrate the tools. We can also discuss this at the next Galaxy single-cell working group meeting.